src/thor/trace_route_action.cc

```diff
@@ -164,7 +146,11 @@ std::list<TripLeg> thor_worker_t::route_match(valhalla_request_t& request,
                                     path_infos.end(), *request.options.mutable_locations()->begin(),
                                     *request.options.mutable_locations()->rbegin(),
                                     std::list<valhalla::Location>{}, interrupt);
+    if (trip_path.node_size() == 0)
+      throw std::exception{};
```

Comment:
this might be redundant.  if `FormPath` above cant do the walk, i think it will return `false` which is equivalent to this. you can test it by doing an `edge_walk` request and sending it some random geometry

Reply:
Yep confirmed locally, will remove